setup.cfg

```diff
@@ -1,2 +1,3 @@
 [flake8]
 max-line-length = 90
+extend-ignore = E203, F403, E402
```

Comment:
Is this a permanent proposal to ignore those?

Reply:
These ignores are temporary for now.